src/gsy_e/models/strategy/forward/pv.py

```diff
@@ -0,0 +1,85 @@
+from typing import Dict, TYPE_CHECKING
+
+from gsy_framework.enums import AvailableMarketTypes
+from pendulum import DateTime
+
+from gsy_e.constants import FLOATING_POINT_TOLERANCE
+from gsy_e.models.strategy.energy_parameters.energy_params_eb import (
+    ProductionStandardProfileEnergyParameters)
+from gsy_e.models.strategy.forward import ForwardStrategyBase
+from gsy_e.models.strategy.forward.order_updater import OrderUpdaterParameters
+
+if TYPE_CHECKING:
+    from gsy_e.models.market.forward import ForwardMarketBase
+    from gsy_framework.data_classes import Trade
+    from gsy_e.models.state import PVState
+
+
+class ForwardPVStrategy(ForwardStrategyBase):
+    """
+    Strategy that models a PV that trades with a Standard Solar Profile on the forward
+    exchanges.
+    """
+    def __init__(
+            self, capacity_kW: float,
+            order_updater_parameters: Dict[AvailableMarketTypes, OrderUpdaterParameters]):
+        super().__init__(order_updater_parameters)
+        self._energy_params = ProductionStandardProfileEnergyParameters(capacity_kW)
+
+    @property
+    def state(self) -> "PVState":
+        return self._energy_params.state
+
+    def event_activate(self, **kwargs):
+        self._energy_params.event_activate_energy(self.area)
+
+    def remove_open_orders(self, market: "ForwardMarketBase", market_slot: DateTime):
+        offers = [offer
+                  for offer in market.slot_offer_mapping[market_slot]
+                  if offer.seller == self.owner.name]
+        for offer in offers:
+            market.delete_offer(offer)
+
+    def post_order(self, market: "ForwardMarketBase", market_slot: DateTime):
+        order_rate = self._order_updaters[market][market_slot].get_energy_rate(
+            self.area.now)
+
+        capacity_percent = self._order_updaters[market][market_slot].capacity_percent / 100.0
+        max_energy_kWh = self._energy_params.capacity_kWh * capacity_percent
+        energy_kWh = self._energy_params.get_available_energy_kWh(
+            market_slot, market.market_type)
+
+        posted_energy_kWh = self._energy_params.get_posted_energy_kWh(
+            market_slot, market.market_type)
+        energy_kWh -= posted_energy_kWh
+
+        energy_kWh = min(energy_kWh, max_energy_kWh)
```

Comment:
Minor, but the following way of implementation would look clearer to my eyes. 
```suggestion
        available_energy_kWh = self._energy_params.get_available_energy_kWh(
            market_slot, market.market_type)

        posted_energy_kWh = self._energy_params.get_posted_energy_kWh(
            market_slot, market.market_type)

        energy_kWh = min(available_energy_kWh-posted_energy_kWh, max_energy_kWh)
```

Reply:
Fixed, thanks